mmcls/models/backbones/swin_transformer_v2.py

```diff
@@ -182,13 +182,25 @@ def __init__(self,
         if not isinstance(block_cfgs, Sequence):
             block_cfgs = [deepcopy(block_cfgs) for _ in range(depth)]
 
-        self.embed_dims = embed_dims
+        if downsample:
+            self.embed_dims = 2 * embed_dims
```

Comment:
The `embed_dims` property doesn't match the `embed_dims` argument. It is a little confusing. Simply use the `out_channels` property as the original behavior is enough.

Reply:
done